packages/sdk/__tests__/dom.spec.ts

```diff
@@ -138,14 +139,15 @@ describe('createScript', () => {
         attrs,
         createScriptHook: (url) => {
           const scriptEle = document.createElement('script');
-          scriptEle.src = url;
+          scriptEle.src = scriptHookUrl;
```

Comment:
why has this test changed?

Reply:
This is ensuring that the script.url value is the one set by the createScriptHook. It's testing that if the createScriptHook defines the script.url that value is not overwritten by the createScript function in dom.ts.

Reply:
ahh understood